specification.rst

```diff
@@ -384,6 +384,9 @@ the item type is "book":
       </citation>
     </style>
 
+The output of a ``<text macro="...">`` does not have delimiters from any
```

Comment:
```suggestion
Delimiters from any ancestor delimiting element are not applied within the output of <text macro="..."> element (see delimiter).
```

How about that?

Reply:
I like it (though I think merging the suggestion won't work since it only replaces the first line.